### ISSUE TITLE
Numpy array shape not recognized as subscriptable

### ISSUE THREAD

**Issue by inducer** [PyCQA/pylint#3381]
### Steps to reproduce

```
import numpy as np

def make_circular_point_group(ambient_dim, npoints, radius,
        center=np.array([0., 0.]), func=lambda x: x):
    t = func(np.linspace(0, 1, npoints, endpoint=False)) * (2 * np.pi)
    center = np.asarray(center)
    result = np.zeros((ambient_dim, npoints))
    result[:2, :] = center[:, np.newaxis] + radius*np.vstack((np.cos(t), np.sin(t)))
    return result


def main():
    point_sources = make_circular_point_group(2, 10, 3, func=lambda x: x**1.5)
    print(point_sources.shape[1])
```

### Current behavior

```
$ pylint test.py
(snip)
test.py:14:10: E1136: Value 'point_sources.shape' is unsubscriptable (unsubscriptable-object)
```

### Expected behavior
 
`point_sources.shape` is a tuple, hence subscriptable. I'm expecting no error to be reported.

### pylint --version output

```
pylint 2.4.4
astroid 2.3.3
Python 3.7.6 (default, Dec 19 2019, 09:25:23) 
[GCC 9.2.1 20191130]
```

**Comment by PCManticore** [comment]
CC @hippo91

**Comment by hippo91** [comment]
@inducer thank you for this report.
I'll investigate ASAP.

**Comment by hippo91** [comment]
@inducer the problem you illustrated has been solved through commit 55b43f6755d4839cefaf010eb2ced747d014d459.
So the fix will be published with the version 2.4.0 of `astroid`.

See PR PyCQa/astroid#732 for more details.

**Comment by wmpauli** [comment]
@hippo91 , thanks for fixing. I'm having the same issue. Looks like pypi.org doesn't have this fix yet, but easy to ignore warning.
